contracts/sol6/KyberNetwork.sol

```diff
@@ -666,6 +657,17 @@ contract KyberNetwork is WithdrawableNoModifiers, Utils5, IKyberNetwork, Reentra
             )
         );
 
+        if (tradeData.input.srcAmount > actualSrcAmount) {
```

Comment:
I think can remove the if, bcos it's checked in the `handleChange` function

Reply:
change as request